README.md

```diff
@@ -35,8 +35,41 @@ with intermezzOS, a learning kernel for the x86_64 platform, written in Rust
 
 ## Learning Objectives
 
+* Understand what an OS is.
```

Comment:
understand is a bad verb for learning objectives. let's make these actionable things.

for theoretical concepts, "be able to explain" is better. additionally "identify 3 abstractions", "compare and contrast X and Y", "describe the tradeoffs with Z"

we should also have actionable code things.